exporter/local/fs.go

```diff
@@ -127,7 +127,14 @@ func CreateFS(ctx context.Context, sessionID string, k string, ref cache.Immutab
 		return res
 	}
 
-	outputFS := fsutil.NewFS(src, walkOpt)
+	outputFS, err := fsutil.NewFS(src)
+	if err != nil {
+		return nil, nil, err
+	}
+	outputFS, err = fsutil.NewFilterFS(outputFS, filterOpt)
+	if err != nil {
+		return nil, nil, err
+	}
```

Comment:
What's going on here? I guess this is initializing the filesystem and then wrapping it in a filter? Might be useful to add a comment here to describe what's going on or even just throw it into a function.

Reply:
Reworked the code slightly, and added comments.

Reply:
👍  I also didn't notice that the name of this function was `CreateFS` the first time so LGTM.